src/index.ts

```diff
@@ -64,7 +64,15 @@ class ServerlessOfflineSSM implements Plugin {
     }
 
     const value = this.config.ssm?.[key]
-    return value ? Promise.resolve(value) : getValueFromEnv(key)
+    const promisifiedValue = value
+      ? Promise.resolve(value)
+      : getValueFromEnv(key)
+    
+    if (key.startsWith('/aws/reference/secretsmanager')) {
+      return promisifiedValue.then(JSON.parse).catch(() => promisifiedValue)
+    } else {
```

Comment:
The `else` here is not necessary. This block should read as the following:

```js
if (cond) {
  return
}

return
```

Reply:
I've fixed it, and removed the unnecessary `else`